superset/tags/api.py

```diff
@@ -211,40 +214,21 @@ def bulk_create(self) -> Response:
         """Bulk create tags and tagged objects
         ---
         post:
-          summary: Get all objects associated with a tag
-          parameters:
-          - in: path
-            schema:
-              type: integer
-            name: tag_id
+          summary: Bulk create tags and tagged objects
           requestBody:
             description: Tag schema
             required: true
             content:
               application/json:
                 schema:
-                  type: object
-                  properties:
-                    tags:
-                      description: list of tag names to add to object
-                      type: array
-                      items:
-                        type: string
-                    objects_to_tag:
-                      description: list of object names to add to object
-                      type: array
-                      items:
-                        type: array
+                  $ref: '#/components/schemas/TagPostBulkSchema'
```

Comment:
The tags API and related schemas are in pretty rough shape. I would almost recommend doing a full refactor of these in some future version.